packages/core/package.json

```diff
@@ -49,5 +49,8 @@
     "rimraf": "~3.0.2",
     "tslog": "^3.2.0",
     "typescript": "~4.3.0"
+  },
+  "resolutions": {
+    "@types/node": "^15.14.1"
```

Comment:
@types/node v16 was released recently which causes errors with some the types. I'm not sure if it is an error in the types (the error seems weird), so for now we can stick to types of node 15